.travis.yml

```diff
@@ -1,52 +1,18 @@
 sudo: false
 
-language: python
+services:
+  - docker
 
-cache:
-  apt: true
-  directories:
-  - $HOME/.cache/pip
-  - $HOME/.ccache
-  - $HOME/.pip-cache
-  - $HOME/third
-  - $HOME/nltk_data
-
-dist: xenial
-
-addons:
-  apt:
-    packages:
-      - expect-dev # for unbuffer, see https://github.com/travis-ci/travis-ci/issues/7967
-
-matrix:
-  include:
-    - python: '3.5'
-      env: TOXENV="py-travis"
-
-    - python: '3.6'
-      env: TOXENV="py-travis"
-
-    - python: '3.7'
-      env: TOXENV="py-travis"
-
-    - python: '3.7' # Only runs checks on 3rd party for latest Python distribution
-      env: TOXENV="py-travis-third-party"
-
-    - python: '3.8'
-      env: TOXENV="py-travis"
+python:
+  - '3.5'
+  - '3.6'
+  - '3.7'
+  - '3.8'
 
 before_install:
-  - source tools/travis/pre-install.sh # Checks Java and Python versions.
-  - chmod +x tools/travis/coverage-pylint.sh
-  - chmod +x tools/travis/third-party.sh
-
-install:
-  - source tools/travis/install.sh
-  #- source tools/travis/third-party.sh
-  - pip install tox
+  - docker build --tag nltk .
```

Comment:
Putting the Dockerfile in the project root directory sends a large build context to the docker daemon.

Putting it in `docker/` or something like that will avoid that.